src/ServiceControl.Monitoring/Infrastructure/ReportThroughputHostedService.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace ServiceControl.Monitoring
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using NServiceBus;
+    using NServiceBus.Metrics;
+    using ServiceControl.Monitoring.Infrastructure.Api;
+
+    class ReportThroughputHostedService(ILogger<ReportThroughputHostedService> logger, IMessageSession session, IEndpointMetricsApi endpointMetricsApi, TimeProvider timeProvider) : BackgroundService
+    {
+        protected override async Task ExecuteAsync(CancellationToken cancellationToken)
+        {
+            logger.LogInformation($"Starting {nameof(ReportThroughputHostedService)}");
+
+            try
+            {
+                //await Task.Delay(DelayStart, cancellationToken);
```

Comment:
why was this removed?

Reply:
that can be removed altogether - left over from copy/paste